server/statistics/metrics.go

```diff
@@ -170,6 +170,14 @@ var (
 			Name:      "flow_queue_status",
 			Help:      "Status of the hotspot flow queue.",
 		}, []string{"type"})
+
+	hotPeerSummary = prometheus.NewGaugeVec(
```

Comment:
Will it  move all hot perr summary from hot region scheduler to statustucs?

Reply:
> will the metrics works well if hot regions scheduler is disable.

The metrics don't have any relationship with the hot scheduler. 
> Will it move all hot perr summary from hot region scheduler to statistics?

I don't get it.